worlds/witness/data/static_locations.py

```diff
@@ -488,3 +498,13 @@ def get_event_name(entity_hex: str) -> str:
 for loc in ALL_LOCATIONS_TO_IDS:
     area = static_witness_logic.ENTITIES_BY_NAME[loc]["area"]["name"]
     AREA_LOCATION_GROUPS.setdefault(area, set()).add(loc)
+
+POSSIBLE_LOCATIONS |= GENERAL_LOCATIONS
+POSSIBLE_LOCATIONS.update(EXTRA_LOCATIONS)
+
+door_shuffle_line_gen = (line for line in utils.get_complex_doors())
```

Comment:
Import this differently

Reply:
Nevermind, this is now in scope of https://github.com/ArchipelagoMW/Archipelago/pull/4297, making it out of scope for this